### ISSUE TITLE
ndcli -d displays misleading "DEBUG - Username: " info

### ISSUE THREAD

**Issue by miesi** [1and1/dim#139]
The effective user comes from the cookie, but it is the user displayed from either the commandline or .ndclirc.

Please display the effective session user (or both) in

```
$ ndcli create rr dhcp-10-78-48-115.dhcp.server.lan. a 10.78.48.115 -c dhcp-pool -d
DEBUG - Dim server URL: https://dim.server.lan/dim
DEBUG - Username: <sessionuser>
```

**Comment by miesi** [comment]
check if fixed with NDCLI_USER option (#156)

**Comment by Srudush** [comment]
That output is correct in so far, as it represents the requested username. When ndcli is using a different cookie with a different username, there is no way to check that apart from asking the server, which username the cookie belongs to.

There are 3 ways here
1. We leave it as it is.
2. We return the username with every request made (could break clients)
3. We make a new call to check the username.

From my point of view, we should leave it as it is for now.
